src/snowflake/cli/plugins/spcs/image_registry/manager.py

```diff
@@ -5,6 +5,14 @@
 import requests
 from click import ClickException
 from snowflake.cli.api.sql_execution import SqlExecutionMixin
+from snowflake.connector.cursor import DictCursor
+
+
+class NoRepositoriesViewableError(ClickException):
+    def __init__(
+        self, msg: str = "Current role does not have view access to any repositories"
```

Comment:
We could not differentiate whether the empty result is due to no read access or no image repository at all. Could we just simply say "No image repository found."

Reply:
good point, changed error message